### PR TITLE
backend: increase SR client timeout and limit concurrency when gettin…

### PR DESCRIPTION
**Pull request by bojand**
…g all schemas and subjects

### CODE REVIEW

File: backend/pkg/schema/client.go [hunk 1]
```diff
@@ -697,9 +697,18 @@ func (c *Client) GetSchemasIndividually(ctx context.Context, showSoftDeleted boo
...
+	guard := make(chan struct{}, 10)
+
 	// Describe all subjects concurrently one by one
 	for _, subject := range subjectsRes.Subjects {
+		guard <- struct{}{}
+
 		go func(s string) {
+			defer func() {
+				<-guard
+			}()
```
Comment (weeco) [inline backend/pkg/schema/client.go:710]: Can we refactor to use errgroup with SetLimit here?

File: backend/pkg/schema/client.go [hunk 2]
```diff
@@ -52,7 +52,7 @@ func newClient(cfg config.Schema) (*Client, error) {
 		SetHeader("Accept", "application/vnd.schemaregistry.v1+json").
 		SetHeader("Content-Type", "application/vnd.schemaregistry.v1+json").
 		SetError(&RestError{}).
-		SetTimeout(5 * time.Second)
+		SetTimeout(30 * time.Second)
```
Comment (weeco) [inline backend/pkg/schema/client.go:55]: Why 30s? Seems like a very long time for a single request to be made. I think not even the frontend would wait for such a long time for the response.

File: backend/pkg/schema/client.go (hunk 1 above)
Comment (bojand) [reply backend/pkg/schema/client.go]: Updated.

File: backend/pkg/schema/client.go (hunk 2 above)
Comment (bojand) [reply backend/pkg/schema/client.go:57]: We were seeing request timeouts to SR when trying to fetch data from `.../versions` endpoint.
```
... context deadline exceeded (Client.Timeout exceeded while awaiting headers)
``` 
Although could be due to the amount of requests we were sending to SR as well.

**Review by weeco — APPROVED** [review]

**Pull request merged**